ngraph/src/ngraph/attribute_visitor.hpp

```diff
@@ -123,7 +123,9 @@ namespace ngraph
         {
             AttributeAdapter<AT> adapter(value);
             start_structure(name);
-            on_adapter(get_name_with_context(), adapter);
+            // todo: it's workaround, remove after fixing #35906
+            //            on_adapter(get_name_with_context(), adapter);
+            on_adapter(name, adapter);
```

Comment:
Do you use this line in order to remove `attr` prefix?

Can you also check, I guess that attribute visitors can work for v0 proposal too. And we can remove creator/convertor in this case for proposal v0.

Reply:
>Do you use this line in order to remove attr prefix?

yes.

>Can you also check, I guess that attribute visitors can work for v0 proposal too. 

I checked it worked. After removing creator networks I have work. Only some tests fails because visitor don't have setting default values if they are not specified in xml. I will add this and will push these changes.

Reply:
Pushed the latest version without creator. Please review.

Reply:
I see that we still have Creator for v0::Proposal (https://github.com/openvinotoolkit/openvino/pull/1448/files#diff-991e8afdea64d14d7b46a914ac98636eR602) can you remove it too?

Reply:
Sorry, I am not right. It is ok.

Reply:
Removed convertor for v0::Propoal.